daft/analytics.py

```diff
@@ -62,21 +64,38 @@ def _build_segment_batch_payload(
     }
 
 
+def _enable_analytics() -> None:
```

Comment:
Ditto above, let's not make this affect global state and `_enable_analytics()` can just be a method on the analytics client.

Reply:
Updated.